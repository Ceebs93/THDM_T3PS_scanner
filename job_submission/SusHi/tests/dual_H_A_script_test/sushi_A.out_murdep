# mh =   0.89674515E+03 GeV
# muR/GeV - sigma(LO)/pb - sigma(NLO)/pb- sigma(NNLO)/pb - sigma(N3LO)/pb
   44.837257607564830       0.25463697266332885       0.27165721751550909       0.19288612131038796        0.0000000000000000     
   46.950372457193495       0.25031783510075944       0.27073747323199232       0.19702499132687129        0.0000000000000000     
   49.163075341549984       0.24610766418186911       0.26976282572638205       0.20093159198806770        0.0000000000000000     
   51.480059700114225       0.24200282452031255       0.26873738142649883       0.20461660259247183        0.0000000000000000     
   53.906240167354227       0.23799983293697405       0.26766499307989861       0.20809017693720916        0.0000000000000000     
   56.446762997324726       0.23409534685662717       0.26654927425956865       0.21136197491286357        0.0000000000000000     
   59.107016979561870       0.23028616057061352       0.26539361622559032       0.21444118776235110        0.0000000000000000     
   61.892644869428473       0.22656919772217859       0.26420120156862759       0.21733656337520268        0.0000000000000000     
   64.809555357154579       0.22294150509376648       0.26297501740372586       0.22005642987425755        0.0000000000000000     
   67.863935600961668       0.21940024673941699       0.26171786766541610       0.22260871781427444        0.0000000000000000     
   71.062264350854718       0.21594269844068081       0.26043238457081158       0.22500098108231675        0.0000000000000000     
   74.411325690919696       0.21256624246589392       0.25912103931193808       0.22724041658340599        0.0000000000000000     
   77.918223429275315       0.20926836261401541       0.25778615203361283       0.22933388278907868        0.0000000000000000     
   81.590396166202709       0.20604663952550858       0.25642990114869096       0.23128791722107073        0.0000000000000000     
   85.435633072413580       0.20289874624395915       0.25505433203842842       0.23310875293733546        0.0000000000000000     
   89.462090410925825       0.19982244401321289       0.25366136518195587       0.23480233408298384        0.0000000000000000     
   93.678308837591047       0.19681557829586427       0.25225280375545744       0.23637433056444113        0.0000000000000000     
   98.093231516970533       0.19387607499986545       0.25083034073850630       0.23783015190115020        0.0000000000000000     
   102.71622309198598       0.19100193690091563       0.24939556556214815       0.23917496030548391        0.0000000000000000     
   107.55708954758353       0.18819124024910708       0.24794997033068833       0.24041368303811017        0.0000000000000000     
   112.62609901054181       0.18544213154905720       0.24649495564672785       0.24155102408291251        0.0000000000000000     
   117.93400352954546       0.18275282450346952       0.24503183606677459       0.24259147518262580        0.0000000000000000     
   123.49206188172259       0.18012159711070408       0.24356184521271645       0.24353932627364211        0.0000000000000000     
   129.31206345401998       0.17754678890755879       0.24208614056257455       0.24439867535590826        0.0000000000000000     
   135.40635325007383       0.17502679834901191       0.24060580794222614       0.24517343783149298        0.0000000000000000     
   141.78785807561698       0.17256008031721015       0.23912186573820371       0.24586735534323712        0.0000000000000000     
   148.47011395796778       0.17014514375246412       0.23763526885021158       0.24648400414285354        0.0000000000000000     
   155.46729485775836       0.16778054939947185       0.23614691240066177       0.24702680301597890        0.0000000000000000     
   162.79424273380556       0.16546490766240732       0.23465763521727878       0.24749902078991248        0.0000000000000000     
   170.46649902489565       0.16319687656290829       0.23316822310368529       0.24790378344815972        0.0000000000000000     
   178.50033761525904       0.16097515979535729       0.23167941191182223       0.24824408087436228        0.0000000000000000     
   186.91279935366151       0.15879850487419628       0.23019189042907534       0.24852277324679523        0.0000000000000000     
   195.72172819932865       0.15666570136833005       0.22870630309208356       0.24874259710328434        0.0000000000000000     
   204.94580907137598       0.15457557921797094       0.22722325253836734       0.24890617109493357        0.0000000000000000     
   214.60460748202675       0.15252700712955555       0.22574330200309861       0.24901600144304536        0.0000000000000000     
   224.71861103768782       0.15051889104293384       0.22426697758512948       0.24907448713456976        0.0000000000000000     
   235.30927289590886       0.14855017267484943       0.22279477036477496       0.24908392483153741        0.0000000000000000     
   246.39905727040588       0.14661982812107927       0.22132713840417861       0.24904651354969831        0.0000000000000000     
   258.01148708067075       0.14472686652474956       0.21986450863066909       0.24896435909933967        0.0000000000000000     
   270.17119384723611       0.14287032880506476       0.21840727860981229       0.24883947830542708        0.0000000000000000     
   282.90396993843444       0.14104928644227391       0.21695581821557075       0.24867380301894410        0.0000000000000000     
   296.23682327946813       0.13926284031649436       0.21551047120391070       0.24846918393073875        0.0000000000000000     
   310.19803463984056       0.13751011959768011       0.21407155669578390       0.24822739419851694        0.0000000000000000     
   324.81721762065882       0.13579028068417898       0.21263937057502266       0.24795013289699927        0.0000000000000000     
   340.12538146905348       0.13410250618746131       0.21121418680631790       0.24763902830066917        0.0000000000000000     
   356.15499685294822       0.13244600396074213       0.20979625867811691       0.24729564100799706        0.0000000000000000     
   372.94006473570033       0.13082000616933848       0.20838581997496072       0.24692146691551028        0.0000000000000000     
   390.51618849670228       0.12922376840072780       0.20698308608349270       0.24651794004959127        0.0000000000000000     
   408.92064945092318       0.12765656881237888       0.20558825503606706       0.24608643526338517        0.0000000000000000     
   428.19248592757594       0.12611770731538330       0.20420150849555990       0.24562827080584199        0.0000000000000000     
   448.37257607564817       0.12460650479324392       0.20282301268638095       0.24514471077110264        0.0000000000000000     
   469.50372457193527       0.12312230235142529       0.20145291927042441       0.24463696742917163        0.0000000000000000     
   491.63075341549995       0.12166446059952744       0.20009136617701617       0.24410620345111300        0.0000000000000000     
   514.80059700114214       0.12023235896209210       0.19873847838486516       0.24355353402825078        0.0000000000000000     
   539.06240167354213       0.11882539501907148       0.19739436866333451       0.24298002889654055        0.0000000000000000     
   564.46762997324743       0.11744298387145531       0.19605913826996169       0.24238671426410369        0.0000000000000000     
   591.07016979561888       0.11608455753344288       0.19473287761076183       0.24177457465159877        0.0000000000000000     
   618.92644869428432       0.11474956434893685       0.19341566686414627       0.24114455464842646        0.0000000000000000     
   648.09555357154602       0.11343746843138794       0.19210757657059244       0.24049756058890528        0.0000000000000000     
   678.63935600961679       0.11214774912592931       0.19080866819005227       0.23983446215244419        0.0000000000000000     
   710.62264350854741       0.11087990049279176       0.18951899462892838       0.23915609389147255        0.0000000000000000     
   744.11325690919773       0.10963343081104460       0.18823860073834003       0.23846325669066976        0.0000000000000000     
   779.18223429275406       0.10840786210175218       0.18696752378529721       0.23775671916086352        0.0000000000000000     
   815.90396166202720       0.10720272966968354       0.18570579389830383       0.23703721897077390        0.0000000000000000     
   854.35633072413520       0.10601758166275382       0.18445343448881993       0.23630546411961925        0.0000000000000000     
   894.62090410925759       0.10485197864841896       0.18321046264992882       0.23556213415343827        0.0000000000000000     
   936.78308837590976       0.10370549320628147       0.18197688953347610       0.23480788132783439        0.0000000000000000     
   980.93231516970457       0.10257770953571364       0.18075272070687237       0.23404333171970532        0.0000000000000000     
   1027.1622309198599       0.10146822307946234       0.17953795649006227       0.23326908629048665        0.0000000000000000     
   1075.5708954758356       0.10037664016183820       0.17833259227476189       0.23248572189869404        0.0000000000000000     
   1126.2609901054172        9.9302577637393780E-002  0.17713661883111889       0.23169379228516609        0.0000000000000000     
   1179.3400352954550        9.8245662555513438E-002  0.17595002258810763       0.23089382899162272        0.0000000000000000     
   1234.9206188172261        9.7205531836618148E-002  0.17477278590630368       0.23008634225532992        0.0000000000000000     
   1293.1206345402002        9.6181831960303540E-002  0.17360488733407795       0.22927182186164533        0.0000000000000000     
   1354.0635325007374        9.5174218664913421E-002  0.17244630185006685       0.22845073796013401        0.0000000000000000     
   1417.8785807561690        9.4182356658071384E-002  0.17129700109266227       0.22762354184596356        0.0000000000000000     
   1484.7011395796783        9.3205919337715451E-002  0.17015695357721616       0.22679066670817172        0.0000000000000000     
   1554.6729485775841        9.2244588523203747E-002  0.16902612490162455       0.22595252834633200        0.0000000000000000     
   1627.9424273380575        9.1298054196076353E-002  0.16790447794090729       0.22510952585705613        0.0000000000000000     
   1704.6649902489569        9.0366014250080631E-002  0.16679197303137608       0.22426204229171096        0.0000000000000000     
   1785.0033761525906        8.9448174250081849E-002  0.16568856814494104       0.22341044528665166        0.0000000000000000     
   1869.1279935366156        8.8544247199502074E-002  0.16459421905408339       0.22255508766721271        0.0000000000000000     
   1957.2172819932871        8.7653953315942126E-002  0.16350887948798593       0.22169630802663792        0.0000000000000000     
   2049.4580907137583        8.6777019814661016E-002  0.16243250128028996       0.22083443128107033        0.0000000000000000     
   2146.0460748202659        8.5913180699598871E-002  0.16136503450892040       0.21996976920167313        0.0000000000000000     
   2247.1861103768770        8.5062176561644706E-002  0.16030642762839567       0.21910262092489388        0.0000000000000000     
   2353.0927289590891        8.4223754383863522E-002  0.15925662759501780       0.21823327344184557        0.0000000000000000     
   2463.9905727040596        8.3397667353410060E-002  0.15821557998531541       0.21736200206771913        0.0000000000000000     
   2580.1148708067058        8.2583674679867047E-002  0.15718322910809174       0.21648907089211117        0.0000000000000000     
   2701.7119384723619        8.1781541419759921E-002  0.15615951811041223       0.21561473321109798        0.0000000000000000     
   2829.0396993843451        8.0991038307007573E-002  0.15514438907784769       0.21473923194185471        0.0000000000000000     
   2962.3682327946817        8.0211941589081903E-002  0.15413778312926979       0.21386280002057595        0.0000000000000000     
   3101.9803463984063        7.9444032868657172E-002  0.15313964050648393       0.21298566078442394        0.0000000000000000     
   3248.1721762065890        7.8687098950539977E-002  0.15214990065896578       0.21210802833819015        0.0000000000000000     
   3401.2538146905358        7.7940931693679091E-002  0.15116850232395423       0.21123010790632807        0.0000000000000000     
   3561.5499685294831        7.7205327868064430E-002  0.15019538360214230       0.21035209617098638        0.0000000000000000     
   3729.4006473570039        7.6480089016330313E-002  0.14923048202919131       0.20947418159663567        0.0000000000000000     
   3905.1618849670235        7.5765021319887757E-002  0.14827373464328469       0.20859654474186318        0.0000000000000000     
   4089.2064945092325        7.5059935469417235E-002  0.14732507804892425       0.20771935855887447        0.0000000000000000     
   4281.9248592757604        7.4364646539559917E-002  0.14638444847716325       0.20684278868122657        0.0000000000000000     
   4483.7257607564788        7.3678973867652695E-002  0.14545178184245763       0.20596699370028335        0.0000000000000000     
