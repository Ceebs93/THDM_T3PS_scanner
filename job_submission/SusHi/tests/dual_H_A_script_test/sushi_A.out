# Date: 05.05.2019 at 13:44:34                     #
# For the reconstructed input file, check below.   #
#--------------------------------------------------#
# Please cite the following papers (for this run):
# Harlander:2012pb    
# Harlander:2016hcx   
# Harlander:2002wh    
# Harlander:2003ai    
# Aglietti:2004nj     
# Bonciani:2010ms     
# Eriksson:2009ws     
# Harlander:2005rq    
# Chetyrkin:2000yt    
#--------------------------------------------------#
Block SUSHIinfo
         1   1.6.1        # SusHi version
         5   1.7.0        # 2HDMC version
Block SUSHIggh # Bon appetit 
         1     2.45144711E-01   # ggh XS in pb                      
       101     8.58276271E-04   # +/- integ. error: ggh XS in pb                      
       102    -9.62115916E-03   # - from muR variation: ggh XS in pb                      
       103     3.93921406E-03   # + from muR variation: ggh XS in pb                      
# within [  0.50000000E+00,  0.20000000E+01]*SCALES(1)*MASSOUT(1).
Block SUSHIbbh # Bon appetit 
         1     2.66523816E-04   # bbh XS in pb                  
       101     3.82172973E-08   # +/- integr. error: bbh XS in pb                  
Block XSGGH # ggh xsec in pb (w/o EW) (+/- integ.err.)
         1     1.24606505E-01   # LO w/ NLO PDFs
         2     2.02823013E-01   # NLO
        21     2.11510206E-01   # NLO gg
        22    -8.69891770E-03   # NLO qg
        23     1.17242909E-05   # NLO qq
       101     6.99110759E-08   # +/-: LO w/ NLO PDFs
       102     1.85259525E-05   # +/-: NLO
       121     1.33810665E-05   # +/-: NLO gg
       122     1.28124046E-05   # +/-: NLO qg
       123     1.62116398E-08   # +/-: NLO qq
Block XSGGHEFF # ggh xsec in heavy top limit (from ggh@nnlo) (+/- integ.err.)
         1     2.33536062E-01   # NLO
       101     3.02055041E-05   # +/-: NLO
        19     2.32842158E-01   # NLO'
       191     2.89225811E-05   # +/-: NLO'
         2     2.75857760E-01   # NNLO
       102     8.57544502E-04   # +/-: NNLO
         4     0.00000000E+00   # electroweak factor
Block XSBBH # bbh xsec in pb (+/- integ.err.)
         1     2.51073282E-04   # LO
       101     1.57905962E-09   # +/-: LO
         2     2.46083328E-04   # NLO
       102     1.33821747E-09   # +/-: NLO
         3     2.66523816E-04   # NNLO
       103     3.82172973E-08   # +/-: NNLO
Block HGGSUSY # couplings of 1.  CP-odd H to 3. gen.
       101     9.09090909E-01   # g_t^phi
       201     1.10000000E+00   # g_b^phi
Block BBHREWEIGHT # top and bottom reweighting factors
         1     9.09090909E-01   # g_t
         2     1.10000000E+00   # g_b
Block MASSOUT
         1     8.96745152E+02   # Mphi in GeV
         5     4.75000000E+00   # m_b(m_b), MSbar
         6     1.72500000E+02   # m_t(pole)
        23     9.11534900E+01   # m_Z
        24     8.03912615E+01   # m_W
        36     8.96745152E+02   # 1.  CP-odd Higgs mass in GeV
Block 2HDMCOUT # 2HDMC results for
         1    1   # stability, 0=failed, 1=ok
         2    0   # perturbativity, 0=failed, 1=ok
         3    0   # unitarity, 0=failed, 1=ok
         7     5.79698077E-03   # h width in GeV
         8     2.99120033E+02   # H width in GeV
         9     5.22315753E+01   # A width in GeV
Block INTERNALMASSES # Masses in GeV
        50     4.75000000E+00   # m_b(m_b), MSbar
        51     2.93243657E+00   # m_b(mu_R) MSbar
        52     4.75000000E+00   # m_b(pole)
        53     4.75000000E+00   # m_b used for internal masses
        54     4.75000000E+00   # m_b for bottom Yukawa
Block ALPHASOUT # values from LHAPDF set
         2     9.65205073E-02   # alpha_s(muRggh) @ NLO
        11     8.78611463E-02   # alpha_s(muRbbh)
         3     9.52042594E-02   # alpha_s(muRggh) @ NNLO
       101     1.35008398E-01   # alpha_s(Mz) @ LO  
       102     1.20006952E-01   # alpha_s(Mz) @ NLO 
       103     1.20006952E-01   # alpha_s(Mz) @ NNLO
#--------------------------------------------------#
# Reconstructed input file of this run             #
#--------------------------------------------------#
Block SUSHI
         1      2   # Chosen model: 0=SM, 1=MSSM, 2=2HDM, 3=NMSSM
         2     21   # 11=h, 12=H, 21=A              
         3      0   # Particle collider: 0=pp, 1=ppbar
         4     1.30000000E+04   # center-of-mass energy in GeV
         5      2   # Order for ggh
         6      2   # Order for bbh
         7      1   # Electroweak contributions to ggh
        11      1   # [1/0] do/not expand C1 perturbatively
        19      0   # 0 = silent mode of SusHi, 1 = normal output
        20      0   # ggh@nnlo subprocesses: 0=all, 10=ind. contributions
        21      0   # bbh@nnlo subprocesses: 0=all
Block GGHMT
        -1      3   # factor out exact LO result at LO(=0)/NLO(=1)/etc.
         0     -1   # expand through 1/mt^n at LO [-1=exact]
         1      0   # expand through 1/mt^n at NLO
        11      0   # expand gg through 1/mt^n at NLO
        12      0   # expand qg through 1/mt^n at NLO
        13      0   # expand qqbar through 1/mt^n at NLO
         2      0   # expand through 1/mt^n at NNLO
        21      0   # expand gg through 1/mt^n at NNLO
        22      0   # expand qg through 1/mt^n at NNLO
        23      0   # expand qqbar through 1/mt^n at NNLO
        24      0   # expand qq through 1/mt^n at NNLO
        25      0   # expand qqprime through 1/mt^n at NNLO
        10      0   # [0/1]: do not/match to high energy limit at NLO
        20      0   # [0/1]: do not/match to high energy limit at NNLO
        30      0   # [0/1]: do not/match to high energy limit at N3LO
Block GGHSOFT # parameters for soft expansion
         1    0  0  0   # NLO: [0/1=n/y] [order] sig(x)/x^[n]
         2    0  0  0   # NNLO: [0/1=n/y] [order] sig(x)/x^[n]
Block 2HDMC
         1      4   # parametrization of 2HDM
         2      2   # 2HDM type
        10      0   # [0/1] ignore validity checks [yes/no]
         3     1.10000000E+00   # tan(beta)
        31     1.25000000E+02   # mh
        32     7.13858000E+02   # mH
        33    -1.48993000E-01   # cos(beta-alpha)
        34     1.85180000E+00   # Z4
        35    -5.03961000E+00   # Z5
        36    -5.87636000E+00   # Z7
Block SMINPUTS
         1     1.27934000E+02   # 1/alpha_em(MZ) SM MSbar
         2     1.16637000E-05   # G_F
         3     1.19000000E-01   # alpha_s(m_Z)
         4     9.11534900E+01   # m_Z(pole)
         5     4.75000000E+00   # m_b(m_b)
         6     1.72500000E+02   # m_t(pole)
Block DISTRIB
         1      0   # total XS                      
Block SCALES
         1     5.00000000E-01   # Renormalization scale muR/mh
         2     5.00000000E-01   # Factorization scale muF/mh
       101     5.00E-01  2.00E+00   # min and max for muR scale uncertainty around SCALES(1)
       102     1.00E-01  1.00E+01   100   # min/max/steps for table output of muR variation
        11     1.00000000E+00   # Renormalization scale muR/mh for bbh
        12     2.50000000E-01   # Factorization scale muF/mh for bbh
Block RENORMBOT
         1      0   # mb used for bottom Yukawa
         4     4.75000000E+00   # Fixed m_b(pole) in GeV
Block PDFSPEC
         1   MMHT2014lo68cl.LHgrid                           # LO   PDF set
        11      0   # LO   PDF set number
         2   MMHT2014nlo68cl.LHgrid                          # NLO  PDF set
        12      0   # NLO  PDF set number
         3   MMHT2014nnlo68cl.LHgrid                         # NNLO PDF set
        13      0   # NNLO PDF set number
Block VEGAS
# parameters for NLO SusHi
         1    10000   # Number of points
         2        5   # Number of iterations
         3        0   # Output format of VEGAS integration
# parameters for ggh@nnlo:
         4     2000   # Number of points
         5        5   # Number of iterations
        14     5000   # Number of points in second run
        15        2   # Number of iterations in second run
         6        0   # Output format of VEGAS integration
# parameters for bbh@nnlo:
         7     2000   # Number of points
         8        5   # Number of iterations
        17     5000   # Number of points in second run
        18        2   # Number of iterations in second run
         9        0   # Output format of VEGAS integration
Block FACTORS
         1     0.00000000E+00   # Factor multiplied with Yukawa of c quark
         2     1.00000000E+00   # t quark
         3     1.00000000E+00   # b quark
#--------------------------------------------------#
# End of SusHi output                              #
#--------------------------------------------------#
